functionary/train/train_lora.py

```diff
@@ -98,6 +98,10 @@ class TrainingArguments(transformers.TrainingArguments):
         default="v2", metadata={"help": "choose prompt template to use for training"}
     )
 
+    use_liger: bool = field(
+        default=False, metadata={"help": "Whether use liger or not"}
```

Comment:
Add Triton supported hardware here for users to be informed of this restriction. [https://github.com/triton-lang/triton?tab=readme-ov-file#compatibility](https://github.com/triton-lang/triton?tab=readme-ov-file#compatibility)